src/styles/blocks/card.scss

```
@@ -0,0 +1,99 @@
+.card {
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid $text-third-color;
+
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background-color: $text-third-color;
+
+  @extend %font-family;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  transition-duration: $duration;
+  transition-property: transform, color;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    color: $text-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    color: $text-second-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+  }
+
+  &__review-title {
+    color: $text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    color: $text-second-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    color: $text-color;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: block;
+    margin-top: 16px;
+    padding: 11px 0;
+
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: 1s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
make a variable for transition-duration
```suggestion
    transition: $transitionDuration;
```